README.md

```diff
@@ -18,10 +18,13 @@ let host = LanguageServerProcessHost(path: executablePath, arguments: [],
     environemnt: [/* the environment your lsp server requires e.g. PATH */])
 
 host.start { (server) in
-    guard let server = server else {
+    guard let server = server as? JSONRPCLanguageServer else {
```

Comment:
Why the cast here?

Reply:
You're right. its not needed, I've force pushed an update.